card/mod/05_email/lib/card/follow_option.rb

```diff
@@ -48,6 +50,15 @@ def follow_opts args
         add_option args, :main
       end
       
+      def follow_test opts={}, &block
+        codename = get_codename opts
+        Card::FollowOption.test[codename] = block
+      end
+      
+      def follow_test_option option, &block
+        Card::FollowOption.test_option[option] = block
+      end
+      
```

Comment:
yeah, the naming was poor.  Sorry.

Actually, if we go your route, we might be able to make it so that *always and *never are the only ones that typically override the follow_test.

Because if *create uses "select_follower_ids" (or maybe "find_follower_ids")?, then it will presumably need to return an array of one id rather than just the id itself.  And then the "follow_test" will be the same as *edited (and both of the voted ones, too).  I like that.